javascript/builtins/Error.json

```diff
@@ -633,6 +683,48 @@
             }
           }
         },
+        "stackTraceLimit": {
+          "__compat": {
+            "tags": [
+              "web-features:javascript"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": "17"
+              },
+              "chrome_android": "mirror",
+              "deno": {
+                "version_added": false
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": "16.17.0"
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "11.1"
```

Comment:
Browserstack 10.1 does not show this - i.e. 11.1 is first version property appears